source/_posts/2023-11-01-php-core-roundup-18.md

```diff
@@ -0,0 +1,149 @@
+
+Dmitry [emailed](https://externals.io/message/121038) PHP Internals mailing list, which led to a lengthy discussion on the merits of the new JIT implementation. After the RFC vote was accepted, Dmitry [plans](https://externals.io/message/121239#121437) to merge the changes to `php-src` and remove the old JIT implementation in the next few days.
+
+### Under discussion: [RFC1867 for non-POST HTTP verbs](https://wiki.php.net/rfc/rfc1867-non-post) by Ilija Tovilo 💜
+
+Now PHP supports the parsing of `multipart/form-data` content type natively, but only for POST requests. If POST request has the `multipart/form-data` content type, the request body is immediately consumed before starting the PHP script and populated into the `$_POST` and `$_FILES` superglobals.
+
+RFC proposes to add a new function `request_parse_body()` to expose the existing functionality to userland so that it may be used for other HTTP verbs.
+
+### Under discussion: [Rounding Integers as int](https://wiki.php.net/rfc/integer-rounding) by Marc Bennewitz
+
+Currently `round()`, `ceil()` and `floor()` functions return float numbers, but when using integers above `2^53` you get unexpected results due to loss of precision.
+
+RFC proposes to perform rounding for a given integer and returning the resulting integer if possible.
+
+### Under discussion: [Unbundle ext/imap, ext/pspell, ext/oci8, and ext/PDO_OCI](https://wiki.php.net/rfc/unbundle_imap_pspell_oci8) by Derick Rethans 💜
+
+RFC proposes to unbundle these extensions: remove them from the PHP source distribution, and move them to PECL.
+
+### Under discussion: [Multibyte for trim function mb_trim, mb_ltrim and mb_rtrim](https://wiki.php.net/rfc/mb_trim) by Yuya Hamada
+
+RFC proposes to add multibyte support for trim functions.
+
+### Under discussion: [Change the edge case of round()](https://wiki.php.net/rfc/change_the_edge_case_of_round) by Saki Takamachi
+
+RFC proposes to change the `round()` behavior, and stop expecting decimal behavior to float point and start expecting floating point to behave as floating point.
+
+
+<br>
+
+## Documentation
+
+While PHP 8.3 is just around the corner, the documentation available on [php.net](https://php.net), requires updating.
+
+George P. Banyard 💜 is tracking the progress for PHP 8.3 related changes in [php/doc-en#2796](https://github.com/php/doc-en/issues/2796), and also triaged issues in the docs and marked several of them as "[good first time](https://github.com/php/doc-en/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)", which are ideal easy picks if you would like to start contributing to PHP docs. You can find the full list on [GitHub](https://github.com/php/doc-en/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
```

Comment:
I just noticed that @Girgias changed the name on her GitHub profile to Gina Peter Banyard. I will make sure to use this name from this point forward, and also in the commit list although the list script would probably use what it finds in the commit text.

Reply:
No worries if it's a bit all over the place, also good reminder that I need to change my Git settings, I had forgotten to do that 😅